core/src/main/java/io/substrait/relation/ConsistentPartitionWindow.java

```diff
@@ -29,7 +29,7 @@ protected Type.Struct deriveRecordType() {
         .struct(
             Stream.concat(
                 initial.fields().stream(),
-                getPartitionExpressions().stream().map(Expression::getType)));
+                getWindowFunctions().stream().map(WindowRelFunctionInvocation::outputType)));
```

Comment:
This is consistent with the language in the [spec](https://substrait.io/relations/physical_relations/#consistent-partition-window-operation)
> Same as Project operator (input followed by each window expression).